Makefile

```diff
@@ -52,7 +52,7 @@ platform_temp = $(subst -, ,$(ARCH))
 GOOS = $(word 1, $(platform_temp))
 GOARCH = $(word 2, $(platform_temp))
 
-BUILDER_IMAGE := golang:1.14
+BUILDER_IMAGE := golang:1.13
```

Comment:
This shouldn't be a change in this PR, it's already in master.  Did you rebase first?

Reply:
Rebased to TOT.